src/main/java/inputTest.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class inputTest {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter your name: ");
+        String nameIn = input.next();
+        String expF = "Alice";
+        String expM = "Bob";
+        if (nameIn.equals(expF) || nameIn.equals(expM)){
```

Comment:
this is really not needed. use the string literals "Alice" and "Bob" in the if.

Reply:
True...